## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 442360`
Commit: `0e3b6286a7dbefb2d0195dcbd1e1c64fc6e2686b`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 29 packages built:</summary>
  <ul>
    <li>bat-extras.batdiff</li>
    <li>bat-extras.batgrep</li>
    <li>bat-extras.batman</li>
    <li>bat-extras.batpipe</li>
    <li>bat-extras.batwatch</li>
    <li>bat-extras.core</li>
    <li>bat-extras.prettybat</li>
    <li>direnv</li>
    <li>fish</li>
    <li>fish-lsp</li>
    <li>fish.doc</li>
    <li>fishMinimal</li>
    <li>fishMinimal.doc</li>
    <li>fishPlugins.bass</li>
    <li>fishPlugins.done</li>
    <li>fishPlugins.fishtape</li>
    <li>fishPlugins.fishtape_3</li>
    <li>fishPlugins.fzf-fish</li>
    <li>fishPlugins.humantime-fish</li>
    <li>fishPlugins.pure</li>
    <li>kitty</li>
    <li>kitty.kitten</li>
    <li>kitty.shell_integration</li>
    <li>kitty.terminfo</li>
    <li>mise</li>
    <li>oh-my-fish</li>
    <li>swaynotificationcenter</li>
    <li>tests.writers.bin.fish</li>
    <li>tests.writers.simple.fish</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 29 packages built:</summary>
  <ul>
    <li>bat-extras.batdiff</li>
    <li>bat-extras.batgrep</li>
    <li>bat-extras.batman</li>
    <li>bat-extras.batpipe</li>
    <li>bat-extras.batwatch</li>
    <li>bat-extras.core</li>
    <li>bat-extras.prettybat</li>
    <li>direnv</li>
    <li>fish</li>
    <li>fish-lsp</li>
    <li>fish.doc</li>
    <li>fishMinimal</li>
    <li>fishMinimal.doc</li>
    <li>fishPlugins.bass</li>
    <li>fishPlugins.done</li>
    <li>fishPlugins.fishtape</li>
    <li>fishPlugins.fishtape_3</li>
    <li>fishPlugins.fzf-fish</li>
    <li>fishPlugins.humantime-fish</li>
    <li>fishPlugins.pure</li>
    <li>kitty</li>
    <li>kitty.kitten</li>
    <li>kitty.shell_integration</li>
    <li>kitty.terminfo</li>
    <li>mise</li>
    <li>oh-my-fish</li>
    <li>swaynotificationcenter</li>
    <li>tests.writers.bin.fish</li>
    <li>tests.writers.simple.fish</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:x: 26 packages failed to build:</summary>
  <ul>
    <li>bat-extras.batdiff</li>
    <li>bat-extras.batgrep</li>
    <li>bat-extras.batman</li>
    <li>bat-extras.batpipe</li>
    <li>bat-extras.batwatch</li>
    <li>bat-extras.core</li>
    <li>bat-extras.prettybat</li>
    <li>direnv</li>
    <li>fish</li>
    <li>fish-lsp</li>
    <li>fish.doc</li>
    <li>fishPlugins.bass</li>
    <li>fishPlugins.done</li>
    <li>fishPlugins.fishtape</li>
    <li>fishPlugins.fishtape_3</li>
    <li>fishPlugins.fzf-fish</li>
    <li>fishPlugins.humantime-fish</li>
    <li>fishPlugins.pure</li>
    <li>kitty</li>
    <li>kitty.kitten</li>
    <li>kitty.shell_integration</li>
    <li>kitty.terminfo</li>
    <li>mise</li>
    <li>oh-my-fish</li>
    <li>tests.writers.bin.fish</li>
    <li>tests.writers.simple.fish</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>fishMinimal</li>
    <li>fishMinimal.doc</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>fishPlugins.bass</li>
    <li>fishPlugins.fishtape_3</li>
    <li>fishPlugins.fzf-fish</li>
    <li>fishPlugins.pure</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>bat-extras.batdiff</li>
    <li>bat-extras.batgrep</li>
    <li>bat-extras.batman</li>
    <li>bat-extras.batpipe</li>
    <li>bat-extras.batwatch</li>
    <li>bat-extras.core</li>
    <li>bat-extras.prettybat</li>
    <li>direnv</li>
    <li>fish</li>
    <li>fish-lsp</li>
    <li>fish.doc</li>
    <li>fishMinimal</li>
    <li>fishMinimal.doc</li>
    <li>fishPlugins.done</li>
    <li>fishPlugins.fishtape</li>
    <li>fishPlugins.humantime-fish</li>
    <li>kitty</li>
    <li>kitty.kitten</li>
    <li>kitty.shell_integration</li>
    <li>kitty.terminfo</li>
    <li>mise</li>
    <li>oh-my-fish</li>
    <li>tests.writers.bin.fish</li>
    <li>tests.writers.simple.fish</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>fish</summary>
<pre>	126 - tmux-autosuggestion.fish (Skipped)
	127 - tmux-bind.fish (Skipped)
	128 - tmux-bind2.fish (Skipped)
	129 - tmux-commandline.fish (Skipped)
	130 - tmux-complete.fish (Skipped)
	131 - tmux-complete2.fish (Skipped)
	132 - tmux-history-search.fish (Skipped)
	133 - tmux-job.fish (Skipped)
	134 - tmux-multiline-prompt.fish (Skipped)
	135 - tmux-prompt.fish (Skipped)
	136 - tmux-signal.fish (Skipped)
	137 - tmux-transient.fish (Skipped)
	138 - tmux-vi-key-bindings.fish (Skipped)

The following tests FAILED:
	 57 - fish_exit.fish (Failed)
Errors while running CTest
FAILED: [code=8] CMakeFiles/fish_run_tests /tmp/nix-build-fish-4.0.6.drv-3/source/build/CMakeFiles/fish_run_tests 
cd /tmp/nix-build-fish-4.0.6.drv-3/source/build &amp;&amp; env FISH_FORCE_COLOR=1 FISH_SOURCE_DIR=/tmp/nix-build-fish-4.0.6.drv-3/source /nix/store/nwwacfh91ycnsipg1fm5rb1l0dpy66rn-cmake-3.31.7/bin/ctest --force-new-ctest-process --output-on-failure --progress
ninja: build stopped: subcommand failed.</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>fishPlugins.bass</summary>
<pre>https://cache.nixos.org/rmx1z3940a5bjw35arfpz31x27ls77p2.narinfo
https://cache.nixos.org/nar/1dgwyh5wpv39qr93ibhflkzr8vvhfn2mk9cjzpvvmpvbc0b1rjij.nar.xz
https://cache.nixos.org/z39srkk8yk0dh33hkjxq49kxxiyi928h.narinfo
https://numtide.cachix.org/z39srkk8yk0dh33hkjxq49kxxiyi928h.narinfo
Running phase: unpackPhase
@nix {&quot;action&quot;:&quot;setPhase&quot;,&quot;phase&quot;:&quot;unpackPhase&quot;}
unpacking source archive /nix/store/rmx1z3940a5bjw35arfpz31x27ls77p2-source
source root is source
Running phase: patchPhase
@nix {&quot;action&quot;:&quot;setPhase&quot;,&quot;phase&quot;:&quot;patchPhase&quot;}
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix {&quot;action&quot;:&quot;setPhase&quot;,&quot;phase&quot;:&quot;updateAutotoolsGnuConfigScriptsPhase&quot;}
Running phase: configurePhase
@nix {&quot;action&quot;:&quot;setPhase&quot;,&quot;phase&quot;:&quot;configurePhase&quot;}
Running phase: buildPhase
@nix {&quot;action&quot;:&quot;setPhase&quot;,&quot;phase&quot;:&quot;buildPhase&quot;}
Running phase: checkPhase
@nix {&quot;action&quot;:&quot;setPhase&quot;,&quot;phase&quot;:&quot;checkPhase&quot;}
error: tool &#x27;make&#x27; not found</pre>
</details>
<details>
<summary>fishPlugins.fishtape_3</summary>
<pre>in function &#x27;@test&#x27; with arguments &#x27;tap = \#\ pass\ 5&#x27;
	called on line 18 of file /private/tmp/nix-build-fishplugin-fishtape-3.0.1.drv-0/source/tests/tap.fish
not ok 30 tap
  ---
    operator: =
    expected: 
    actual: &#x27;# pass 5&#x27;
    at: /private/tmp/nix-build-fishplugin-fishtape-3.0.1.drv-0/source/tests/tap.fish:18
  ...
not ok 31 tap
  ---
    operator: =
    expected: &#x27;# fail 1&#x27;
    actual: &#x27;&#x27;
    at: /private/tmp/nix-build-fishplugin-fishtape-3.0.1.drv-0/source/tests/tap.fish:19
  ...

1..31
# pass 21
# fail 10</pre>
</details>
</details>

--- a/pkgs/by-name/fi/fish/package.nix
+++ b/pkgs/by-name/fi/fish/package.nix
@@ -234,6 +234,7 @@ stdenv.mkDerivation (finalAttrs: {
     rm tests/pexpects/job_summary.py
     rm tests/pexpects/signals.py
     rm tests/pexpects/fg.py
+    rm tests/checks/fish_exit.fish
   ''
   + lib.optionalString (stdenv.hostPlatform.isAarch64 || stdenv.hostPlatform.isDarwin) ''
     # This test seems to consistently fail on aarch64 and darwin

